src/github.com/getlantern/flashlight/proxiedsites/proxiedsites.go

```diff
@@ -52,6 +56,24 @@ func Configure(cfg *proxiedsites.Config, proxyAddr string) {
 	startMutex.Unlock()
 }
 
+func updateDetour(delta *proxiedsites.Delta) {
+	// TODO: subscribe changes of geolookup and set country accordingly
+	// safe to hardcode here as IR has all detection rules
+	detour.SetCountry("IR")
```

Comment:
Rather than encoding country into detour itself, it may make sense to provide a menu of detection rules from which the caller can pick and choose when initializing detour.  If the caller wants to manage it on a per-country basis, it can.